gnd/src/main/java/com/google/android/gnd/persistence/local/LocalDataStore.java

```diff
@@ -164,6 +164,18 @@
   /** Returns all queued, failed, and completed offline areas from the local data store. */
   Flowable<ImmutableList<OfflineBaseMap>> getOfflineAreasOnceAndStream();
 
+  /** Delete an offline area and its *unique* tiles. */
+  Completable deleteOfflineArea(String offlineAreaId);
```

Comment:
" Delete an offline area and any tiles that are no longer needed." might be clearer here.

Reply:
Done, nice edit, thanks!